src/main/java/me/karakelley/tictactoe/PlayerFactoryImplementation.java

```diff
@@ -0,0 +1,15 @@
+package me.karakelley.tictactoe;
+
+public class PlayerFactoryImplementation implements PlayerFactory{
+  @Override
+  public Player makePlayer(String type, String marker) {
+    Player newPlayer = null;
+
+    if (type.equals("1")) {
+      newPlayer = new HumanPlayer(marker);
+    } else {
+      System.exit(0);
```

Comment:
The PlayerFactory has knowledge about when to exit the game. Couldn't this have been decided earlier? It seems like the minute you get an input that's not 1, you can exit the game, or throw some exception and catch it and then exit the game.

Also, at this point, you only have one type of player, the human player, but you already created a factory and are preparing for other type of players. As Aaron would say, don't overthink it. Do the minimum that needs to be done to meet the acceptance criteria for the given story.